modules/network-observability-dns-tracking.adoc

```diff
@@ -33,12 +33,15 @@ spec:
     ebpf:
       features:
        - DNSTracking           <1>
-      privileged: true         <2>
 ----
 <1> You can set the `spec.agent.ebpf.features` parameter list to enable DNS tracking of each network flow in the web console.
-<2> Note that the `spec.agent.ebpf.privileged` specification value must be `true` for DNS tracking to be enabled.
 
 . When you refresh the *Network Traffic* page, there are new DNS representations you can choose to view in the *Overview* and *Traffic Flow* views and new filters you can apply.
 .. Select new DNS choices in *Manage panels* to display graphical visualizations and DNS metrics in the *Overview*.
 .. Select new choices in *Manage columns* to add DNS columns to the *Traffic Flows* view.
-.. Filter on specific DNS metrics, such as *DNS Id*, *DNS Latency* and *DNS Response Code*, and see more information from the side panel.
\ No newline at end of file
+.. Filter on specific DNS metrics, such as *DNS Id*, *DNS Latency* and *DNS Response Code*, and see more information from the side panel.
+
+[NOTE]
+===
+TCP handshake packets do not have DNS headers. You can filter for flows that have DNS errors using the "DNSError" filter in the *Common* filters and specifying "0" errors. 
```

Comment:
This statement needs to be updated based on what I have told in the demo. Let me know when once you had a chance to look at the video